diff of execution of:
echo 'cat < infile >> out; echo hah >> out; ls -l --time-style=+' | tr ';' '\n' | [shell]
--- expected	2024-08-27 19:27:16.217621641 +0200
+++ found	2024-08-27 19:27:16.236621856 +0200
@@ -1,9 +1,9 @@
 total 8
 -rw-rw-r-- 1 bruno bruno 34  infile
--rw-rw-r-- 1 bruno bruno 38  out
+-rw-r--r-- 1 bruno bruno 38  out
 -r--r--r-- 1 bruno bruno  0  readonly
 
 ======== stderr =======
 
 ======== return =======
 return value: 0
